packages/web-components/src/components/cbp-usa-banner/cbp-usa-banner.tsx

```diff
@@ -14,71 +14,64 @@ export class CbpUsaBanner {
-  @Prop() sx: any = {};
-  
-  /** A custom event emitted when the banner link control is activated. */
-  @Event() bannerLinkClick: EventEmitter;
-  handleClick(e) {
-    e.preventDefault();
+  // /** A custom event emitted when the banner link control is activated. */
+  handleClick() {
+    // e.preventDefault();
     this.open = !this.open;
-    this.bannerLinkClick.emit({
-      host: this.host,
-      open: this.open,
-    });
+    //set aria-expanded if it isn't comine with button
   }
 
   render() {
     return (
       <Host>
-        <img src="../assets/images/us_flag_small.png" height="11" width="16" alt="usa logo"/>Official website of the United State government. 
-        <cbp-link class='cbp-usa-banner-expand' onClick={() => this.handleClick(event)} href='#'>
-          Here is how you know<cbp-icon name='chevron-right'/>
-        </cbp-link>
-        <div class='cbp-usa-banner-content'>
-          <cbp-icon 
-              class='cbp-banner-content-icon'
-              color="var(--cbp-color-info-light)"
-              name='landmark'
-              size="1.25rem" 
-          />  
-          <div>
-            <cbp-typography
-              tag='h5'
-              sx='{"color": "var(--cbp-usa-banner-color)"}'
-            >
-              Official websites use .gov
-            </cbp-typography>
-            <cbp-typography
-              tag='p'
-              sx='{"margin-bottom":0}'
-            >
-              A <b>.gov</b> website belongs to the official government organization in the United States.
-            </cbp-typography>
-          </div>
-          <cbp-icon 
-              class='cbp-banner-content-icon'
-              color="var(--cbp-color-info-light)"
-              name='lock'
-              size="1.25rem" 
-            />
-          <div>
-            <cbp-typography
-              tag='h5'
-              sx='{"color": "var(--cbp-usa-banner-color)"}'
-            >
-              Secure .gov websites use HTTPS
-            </cbp-typography>
-            <cbp-typography
-              tag='p'
-              sx='{"margin-bottom":0}'
-            >
-              A lock(<cbp-icon name='lock' />) or https:// means you've safetly connected to the .gov website. Share sensitive information only on official, secure websites
-            </cbp-typography>
-          </div>
-          <slot>  
-          </slot>
-        </div>
+         <section aria-label="Official website of the United States government">
+           <header class="cbp-banner">
+      
+           <div class="cbp-banner__title">
+              <img src="../assets/images/us_flag_small.png" height="11" width="16" alt="usa logo" />
+              <span>
+                <span>An official website of the United States government.</span>
+                <cbp-button
+                  class='cbp-usa-banner-expand'
+                  fill='ghost'
+                  accessibility-text='open banner content'
```

Comment:
The button already has an accessible label, so you don't need to provide this prop.

Reply:
removed